browser.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2015 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var ChromeDriver = require('./helpers/browser/driver');
+
+/**
+ * @returns {Promise}
+ */
+function construct(URL) {
```

Comment:
it probably makes more sense to pull this into `driver.js` directly, but we can do that in a followup PR if you want